tests/pr_test.go

```diff
@@ -33,6 +33,9 @@ const yamlLocation = "../common-dev-assets/common-go-assets/common-permanent-res
 // Setting "add_atracker_route" to false for VPC and VSI tests to avoid hitting AT route quota, right now its 4 routes per account.
 const add_atracker_route = false
 
+// Setting "service_endpoints" to `private` to check if it is going to pass, since schematics should have access to private network.
```

Comment:
you probably want to now update this comment, as it seems private worked just fine in the schematics test